sched/scheduler/scheduler.go

```diff
@@ -65,24 +65,20 @@ func (s *scheduler) ScheduleJob(job sched.Job) error {
 
 		// Start Running Job
 		s.wg.Add(1)
-		go s.runJob(job, sagaState, nodes)
+		go func() {
+			defer s.wg.Done()
```

Comment:
its just for testing / demo.  Since there is async work being done we want to wait before we verify state.  I updated the comment saying this is for testing only